security/throttling.py

```diff
@@ -43,27 +43,43 @@ def _validate(self, request):
         return count_same_requests <= self.throttle_at
 
 
-class UnsuccessfulLoginThrottlingValidator(ThrottlingValidator):
-
-    def __init__(self, timeframe, throttle_at, description=_('Too many login attempts')):
-        super(UnsuccessfulLoginThrottlingValidator, self).__init__(timeframe, throttle_at, description)
+class LoginThrottlingValidator(ThrottlingValidator):
 
     def _validate(self, request):
         count_same_requests = InputLoggedRequest.objects.filter(
             ip=get_ip(request), path=request.path,
             request_timestamp__gte=timezone.now() - timedelta(seconds=self.timeframe),
-            type=InputLoggedRequest.UNSUCCESSFUL_LOGIN_REQUEST).count()
+            type=self.type).count()
```

Comment:
proč to tady filtrujeme podle typu? Já bych to nedělal je to zbytečné. Ten typ tam dáváme spíš pro administrátorské účely